rotkehlchen/tests/api/test_ignored_actions.py

```diff
@@ -150,3 +154,57 @@ def test_remove_ignored_actions(rotkehlchen_api_server):
         'ledger action': ['1', '2', '3'],
         'trade': ['1', '2', '3'],
     }
+
+
+@pytest.mark.parametrize('number_of_eth_accounts', [0])
+def test_ignore_ledger_actions_in_accountant(rotkehlchen_api_server, accountant):
+    """Test that ignored ledger actions are correctly ignored by the accountant"""
+    ledger_actions_list = [
+        LedgerAction(
+            identifier=1,
+            timestamp=1467279735,
+            action_type=LedgerActionType.INCOME,
+            location=Location.BLOCKCHAIN,
+            amount=FVal(1000),
+            asset=A_ETH,
+            rate=FVal(100),
+            rate_asset=A_USD,
+            link=None,
+            notes=None,
+        ), LedgerAction(
+            identifier=2,
+            timestamp=1467279735,
+            action_type=LedgerActionType.EXPENSE,
+            location=Location.BLOCKCHAIN,
+            amount=FVal(5),
+            asset=A_DAI,
+            rate=None,
+            rate_asset=None,
+            link='foo',
+            notes='boo',
+        ),
+    ]
+
+    # Set the server to ignore second ledger action
+    response = requests.put(
+        api_url_for(
+            rotkehlchen_api_server,
+            "ignoredactionsresource",
+        ), json={'action_type': 'ledger action', 'action_ids': ['2']},
+    )
+    result = assert_proper_response_with_result(response)
+    assert result == {'ledger action': ['2']}
+
+    # Retrieve ignored actions mapping. Should contain 2
+    ignored_actions = accountant.db.get_ignored_action_ids(action_type=None)
+    ignored = []
+    # Call the should_ignore method used in the accountant
+    for action in ledger_actions_list:
+        should_ignore, _ = accountant._should_ignore_action(
+            action=action,
+            action_type='ledger_action',
+            ignored_actionids_mapping=ignored_actions,
+        )
+        ignored.append(should_ignore)
+
+    assert ignored == [False, True]
```

Comment:
This is a simple way of doing this. I was thinking more of a full report like we do in the accounting tests, and then see that only 1 ledger action is taken into account. Since that would test the entire logic.

But this should also be fine.